src/parser/core/modules/GlobalCooldown.js

```diff
@@ -156,16 +156,8 @@ export default class GlobalCooldown extends Module {
 
 		const action = this.data.getAction(gcdInfo.event.ability.guid)
 		if (!action || !action.id) { return }
-		let speedMod = this.speedmod.get(gcdInfo.event.timestamp)
-		let castTime = action.castTime
-
-		// HACK NOTE TODO: Need to properly account for abilities that alter only the cast or recast of attacks.
-		// Thinking of moving this into a module like speedmod, that can be called with a timestamp to grab modified base castTime/cooldown values
-		const HACK_ASTRAL_UMBRAL_SPEED_SCALAR = 0.5
-		if (speedMod <= HACK_ASTRAL_UMBRAL_SPEED_SCALAR) {
-			speedMod /= HACK_ASTRAL_UMBRAL_SPEED_SCALAR
-			castTime *= HACK_ASTRAL_UMBRAL_SPEED_SCALAR
-		}
+		const speedMod = this.speedmod.get(gcdInfo.event.timestamp)
```

Comment:
probably out of scope for this PR, can be done when upgrading GCD - probably want to move this onto speed adjustments

Reply:
Yeah, I was just removing BLM junk and resolving a lint grump about the speedMod value now being a const there. Wasn't aiming to change anything, just clean it up and maintain parity with the previous state. Someone can certainly do more with it/throw it out and redo it in the speed/cd2/gcd/what have you updates that are coming.